blivet/blivet.py

```diff
@@ -145,8 +144,8 @@ def __init__(self, ksdata=None):
 
         # these will both be empty until our reset method gets called
         self.devicetree = DeviceTree(conf=self.config,
-                                     passphrase=self.encryption_passphrase,
-                                     luks_dict=self.__luks_devs)
+                                     passphrase=luks_data.encryption_passphrase,
+                                     luks_dict={})
```

Comment:
I think the ``luks_dict`` parameter should default to ``None`` with ``DeviceTree`` creating the empty dict in such cases.